project/LibDependencies.scala

```diff
@@ -14,7 +14,9 @@ object LibDependencies {
 
   val test: Seq[ModuleID] = Seq(
     "org.scalatest" %% "scalatest" % "3.0.5" % Test,
-    "org.pegdown"   %  "pegdown"   % "1.6.0" % Test
+    "org.pegdown"   %  "pegdown"   % "1.6.0" % Test,
+    "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.0" % Test,
+    "org.jsoup" % "jsoup" % "1.8.1" % Test
```

Comment:
There is more recent compatible versions of the 2 additional libraries added. Can you update them please?